src/Peering/Peering/Exchange/SetAzureExchangePeeringConnectionCommand.cs

```diff
@@ -194,16 +191,13 @@ private PSExchangeConnection UpdateIpV4Prefix()
         /// </returns>
         private PSExchangeConnection UpdateIpV6Prefix()
         {
-            if (this.InputObject is PSExchangeConnection inputObject)
+            if (this.InputObject.GetType().Equals(typeof(PSExchangeConnection)))
```

Comment:
Why use direct type comparsion instead of `is`? Except for extremely rare cases, `is` is what you want.